tools/jenkins_mongo.rb

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env ruby
+
+lib_path = File.expand_path(File.dirname(__FILE__))
+unless $LOAD_PATH.any? {|p| File.expand_path(p) == lib_path}
+  $LOAD_PATH.unshift(lib_path)
+end
+require 'mongo'
+require 'common'
+
+# Turn off debug-mode
+Mongo::Logger.logger.level = Logger::WARN
+
+module BushSlicer
+  class JenkinsMongo
```

Comment:
maybe a better class name or explanation what is this class used for in a comment?

Reply:
Maybe then call it `InstallerLogMongo` or something?

Reply:
@akostadinov it caches the jenkins Flexy-install profile...not Installer log...

Reply:
Cache with what purpose? Cache is to store temporary data for faster search/access. Log is when you store a record of something that happened. In this case it is a record of what clusters have been installed so you can later clean-them up. That's my line of thought.

Reply:
@akostadinov to me the name `JenkinsMongo` is saying it's storing something from Jenkins....the db name is `cache_jenkins`.  What you suggested is more specific, where as as it stands it's more generic.

Reply:
Exactly, it is too generic and it is not clear what and why is it storing. Naming of classes and variables when programming is one of the most important tasks and is not easy. Organizing class hierarchy even more so.